node/src/node_configurator/configurator.rs

```diff
@@ -701,23 +720,15 @@ impl Configurator {
         msg: UiSetConfigurationRequest,
         context_id: u64,
     ) -> MessageBody {
-        let configuration_change_msg_sub_opt = self.configuration_change_msg_sub_opt.clone();
-        let logger = &self.logger;
         debug!(
-            logger,
+            self.logger,
             "A request from UI received: {:?} from context id: {}", msg, context_id
         );
-        match Self::unfriendly_handle_set_configuration(
-            msg,
-            context_id,
-            &mut self.persistent_config,
-            configuration_change_msg_sub_opt,
-            logger,
-        ) {
-            Ok(message_body) => message_body,
+        match self.unfriendly_handle_set_configuration(msg) {
```

Comment:
As Bert pointed out, all the other `unfriendly_handle_xxx` methods return `MessageBody`s; perhaps for consistency this one should as well.